changelog/update-9971-simplify-localization-csv-export

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Simplify localization of CSV exports to use global site language settings from WP Admin
```

Comment:
💭 An idea for updating this changelog entry, to mention a bit more of the "why".

```suggestion
Simplify localization of CSV exports to use user language settings from WP Admin, allowing the CSV export to match the localization of the data presented in the UI.
```

Reply:
That makes sense. I have updated it here [ca2a2fa](https://github.com/Automattic/woocommerce-payments/pull/10003/commits/ca2a2fa2fef43179276c9001ee30e2e5cbe5d530)